autogpts/forge/forge/components/README.md

```diff
@@ -32,21 +32,30 @@ class MyAgent(Agent):
 
 ## Ordering components
 
-The execution order of components is important because the latter ones may depend on the results of the former ones.
+The execution order of components is important because some may depend on the results of the previous ones.
+**By default, components are ordered alphabetically.**
```

Comment:
We should probably resolve this in some other way down the line. Perhaps with a manifest and building a graph